TractorMod/Framework/Config/StandardAttachmentConfig.cs

```diff
@@ -7,7 +7,7 @@ internal class StandardAttachmentsConfig
         public AxeConfig Axe { get; set; } = new();
 
         /// <summary>Configuration for the fertilizer attachment.</summary>
-        public GenericAttachmentConfig Fertilizer { get; set; } = new();
+        public ExtendedDistanceConfig Fertilizer { get; set; } = new() { IncreaseDistance = false };
```

Comment:
For fertilizer and seeding, I felt like it fits the "non-destructive" idea, but as for playing this way all the time, I'm not so sure.  It's a rarely done activity, so personally, I care more about precision than speed.  So I made it something that folks could turn on if they felt differently about it, but left if off by default.